.github/workflows/ci.yml

```diff
@@ -2,27 +2,39 @@
 name: CI
 
 on:
-  - push
   - pull_request
 
 permissions:
   contents: read
 
 jobs:
   rspec:
-    runs-on: ubuntu-20.04
+    name: >-
+      rspec (${{ matrix.ruby }})
+
+    runs-on: ubuntu-latest
+
     strategy:
       fail-fast: false
       matrix:
         ruby:
-          - '3.3'
-          - '3.2'
-          - '3.1'
-          - '3.0'
-          - '2.7'
-          - '2.6'
-          - '2.5'
-          - '2.4'
+          - 3.3
+          - 3.2
+          - 3.1
+        rails:
+          - 6.1 # TODO: move to 7.2
+        include:
+          # Edge
+          - { ruby: 'head', rails: 'edge', allow-fail: true }
+          # Outdated
+          - { ruby: '3.0',  rails: '6.1',  allow-fail: true } # TODO: move to 7.1
+          - { ruby: '2.7',  rails: '6',    allow-fail: true } # RSpec AR Expectations support Rails 7.1 since Ruby 3.0
+          - { ruby: '2.6',  rails: '6',    allow-fail: true }
+          - { ruby: '2.5',  rails: '6',    allow-fail: true }
```

Comment:
I do not think it is a good idea to use `continue-on-error` in these cases.

This setting makes it impossible to notice that a test has failed, which would make the CI meaningless.
I think it's better to consider drop old versions support instead of using this(I think it's better to only suport not EOL versions).

Reply:
> This setting makes it impossible to notice that a test has failed…

Harder to, but not impossible. One still can check for `Test success` or `Test failure` in the steps.

> I think it's better to consider drop old versions support instead of using this(I think it's better to only suport not EOL versions).

I'd like to indeed, but…

1. It's possible for the next _major_ version only.
2. We still have **no access to the settings** to do that 😕

Reply:
> I do not think it is a good idea to use `continue-on-error` in these cases.

I've dropped this for old Rubies until we drop support for them.